webplugin/js/app/km-utils.js

```diff
@@ -201,6 +201,23 @@ Kommunicate = {
     internetStatus: true,
 };
 
+KM_WidgetEvents={
```

Comment:
can you move all analytics related code to a separate file, and keep the minimum code in other files just for calling the respective function.
this will make the code easy to understand

Reply:
for the existing event (i.e. `onMessageSent`) keeping them the same as they are, and for new will create a new file.